build.gradle

```diff
@@ -21,13 +21,14 @@ dependencies {
 	testImplementation 'junit:junit:4.12'
 	testImplementation group: 'net.runelite', name:'client', version: runeLiteVersion
 	testImplementation group: 'net.runelite', name:'jshell', version: runeLiteVersion
+	testImplementation group: 'net.runelite.jogl', name:'jogl-gldesktop-dbg', version: '2.4.0-rc-20220318'
```

Comment:
this should be `compileOnly` and also `testImplementation`

Reply:
Oh? Does that mean I can just it import normally too?

Reply:
Looks like it does, thanks :+1: